catalog-rest-service/src/main/resources/json/schema/entity/services/databaseService.json

```diff
@@ -70,6 +71,9 @@
         },
         {
           "name": "Druid"
+        },
```

Comment:
lets remove this from databaseServices. We shouldn't be publishing any database/tables from Metabase